FakeServer/GraphQL/GraphQLMiddleware.cs

```diff
@@ -20,6 +21,7 @@ public class GraphQLMiddleware
         private readonly bool _authenticationEnabled;
         private readonly string _idFieldName;
         private readonly string[] _allowedTypes = new[] { "application/graphql", "application/json" };
+        private readonly string[] _allowedMethods = new[] { "GET", "POST" };
```

Comment:
Instead of "GET" use HttpMethods.Get etc. Often I also forget this as code had "POST", but you can also change that :)